test_dataloader.py

```diff
@@ -0,0 +1,11 @@
+import cv2
+import matplotlib.pyplot as plt
+import torchvision.transforms as transforms
+
+from dataloader import ChestXrayDataset, ChestXrayTestDataset
+
+if __name__ == "__main__":
+    # dataset = ChestXrayDataset(phase="train", crop=True, transfom=transforms.ToTensor())
+    # print(dataset[0])
+    test_dataset = ChestXrayTestDataset(crop=True, n_per_image=10)
+    print(test_dataset[0])
```

Comment:
A test that requires a human to read outputs (i.e. is not automated) is not ideal. This should be in a `tests` folder, and runnable with something like `pytest` or `unittest`. It should also include some `assert`s that check some important functionality. For example:

```suggestion
def test_dataset() -> None:
    for patch_size in [16, 32, 46, 128]:
        dataset = ChestXrayTestDataset(crop=True, n_per_image=10, patch_size=patch_size)
        assert dataset[0][0].shape = (1, patch_size, patch_size)
        assert dataset[0][1].int64().item() in [0, 1]
```